backend/src/services/mail-verification/send-activation-link/send-link.ts

```diff
@@ -1,12 +1,13 @@
-import path from 'path';
-
 import { ENV } from '@common/enums/app/app';
 import nodemailer from 'nodemailer';
-import hbs from 'nodemailer-express-handlebars';
 
 import { MailActivate } from './constants';
+
 const sendLink = async (email: string, token: string): Promise<void> => {
-  const templatePath = path.join(__dirname, 'templates');
+  const templatePath =
+    '@services/mail-verification/send-activation-link/templates/email-template.ts';
```

Comment:
@setar92 
This won't work because only `tsc-alias` knows how to resolve such path